packages/dapp-high-roller/package.json

```diff
@@ -26,7 +26,7 @@
   "homepage": "https://counterfactual.com",
   "devDependencies": {
     "@stencil/sass": "^0.1.1",
-    "@types/jest": "24.0.9",
```

Comment:
?

Reply:
Running `yarn test` does this automatically. I think its harmless.

Reply:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll It's valid. We were using a wrong `types` package for Jest. (Using Jest 23 with typings for Jest 24).